scripts/e2e-test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+set -e
+
+SCRIPTDIR=$(dirname "$0")
```

Comment:
Change to 
`SCRIPTDIR="$(dirname "$(realpath "$0")")"`

Reply:
```suggestion
SCRIPTDIR="$(realpath "$(dirname "$0")")"
```

Reply:
fixed. thanks 👍